cardano-node/src/Cardano/Node/Parsers.hs

```diff
@@ -253,6 +258,15 @@ parseVrfKeyFilePath =
         <> completer (bashCompleter "file")
     )
 
+-- TODO revisit because it sucks
+parseSnapshotInterval :: Parser SnapshotInterval
```

Comment:
Nice comment :smile: . I would hijack this: https://github.com/input-output-hk/cardano-node/blob/e32ca3408aa8f77b5d1bdbbd42634e1a048f8a97/cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs#L2443
and this: https://github.com/input-output-hk/cardano-node/blob/e32ca3408aa8f77b5d1bdbbd42634e1a048f8a97/cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs#L2535

then you can just use `Opt.option` and all is right in the world.

Reply:
Omg, I forgot about that one 🤦  :) 
Thanks for the hint, I will have a look right away.